plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/util/ObjectStoreLocationProvider.java

```diff
@@ -40,12 +43,14 @@ public class ObjectStoreLocationProvider
     private static final int ENTROPY_DIR_DEPTH = 3;
     private final String storageLocation;
     private final String context;
+    private final boolean includePartitionPaths;
 
     public ObjectStoreLocationProvider(String tableLocation, Map<String, String> properties)
     {
         this.storageLocation = stripTrailingSlash(dataLocation(properties, tableLocation));
         // if the storage location is within the table prefix, don't add table and database name context
         this.context = storageLocation.startsWith(stripTrailingSlash(tableLocation)) ? null : pathContext(tableLocation);
+        this.includePartitionPaths = propertyAsBoolean(properties, WRITE_OBJECT_STORE_PARTITIONED_PATHS, WRITE_OBJECT_STORE_PARTITIONED_PATHS_DEFAULT);
```

Comment:
nit: skimming through the code, I see that the previous maintainers opted for `TableProperties.` approach. Consider a consistent referencing scheme within the class for the constants from `TableProperties` class.